Fix: warning: stray \ before /

edmundlod opened the pull request:
Removed backslashes in `grep -E` lines

edmundlod committed:
Removed \ in 'grep -E' arguments.

pbiering reviewed (approved):
looks ok

jsarenik commented:
To me it seems like too many changes in one commit. Removing the shebang is a showstopper.

jsarenik reviewed (changes requested):
1. Return of the Shebang
2. Get rid of unrelated double-quote removals that were added for sanity and to make shellcheck happy
3. Remove indentation changes

lquidfire commented:
Thank you very much for your eyes and for taking the time to comment what changes are required, @jsarenik! 

I apologise for the (weird!) issues with this PR - I am not even sure how my branch got so messed up (no shebang??).

I hope that things look okay now. All I wanted was to help.

jsarenik commented on include/despf.inc.sh:
```diff
@@ -88,10 +88,10 @@ printip() {
+      get_addr $TYPE $1 | while read ip ; do 
+        addr="${3}${ip}"
+        echo $addr | printip $2;
+      done
```
Spaces replaced TABs ^^ no matter which is the best, try minimal change

lquidfire replied:
Now that got me down a bit of a rabbit hole.. I have learned something new and will change tabs to spaces.

lquidfire replied:
When I look at the original it seems to me that, for example, line 90 has a tab (4 spaces wide), and then 2 spaces. Is that correct? Am I seeing it incorrectly? I am confused..

jsarenik commented on include/despf.inc.sh:
```diff
@@ -236,10 +236,10 @@ despfit() {
-  if grep -E '^[\?\~\-]' $outputfile  ; then
-	  cat $outputfile
+  if grep -E '^[?~-]' $outputfile  ; then
+      cat $outputfile
```
Here again.

jsarenik commented on include/despf.inc.sh:
```diff
@@ -88,7 +88,7 @@ printip() {
 # fec0::1
 dea() {
   for TYPE in A AAAA; do
-	  get_addr $TYPE $1 | while read ip ; do 
```
Here

jsarenik commented on include/despf.inc.sh:
```diff
@@ -236,10 +236,10 @@ despfit() {
 	  cat $outputfile
   else
-	  sort -u $outputfile
+      sort -u $outputfile
```
And here

jsarenik reviewed (changes requested):
Still two more

edmundlod committed:
Changes spaces to "tab(4) + 2 spaces"

edmundlod committed:
Merge branch 'spf-tools:master' into master

edmundlod committed:
Add Debian packaging and CI workflow

edmundlod committed:
ci: dispatch to apt repo after successful .deb build

edmundlod committed:
docs: add Debian package note

edmundlod committed:
ci: add RPM build workflow and spec file

Builds an AlmaLinux 10 RPM on tagged releases and dispatches to the
edmundlod/rpm repository.

edmundlod committed:
Release 20260411

edmundlod committed:
ci: update actions to Node.js 24
